.env.example

```diff
@@ -1,2 +0,0 @@
-# DOCUMENT YOUR REQUIRED ENV VARIABLES HERE
```

Comment:
It is actually best practice to keep this file. If you go back to the cloud storage article you will find an env.example file